### ISSUE TITLE
ANPR for indian number plates gives very low accuracy

### ISSUE THREAD

**Issue by HemanthKumarAchanta** [open-edge-platform/geti#364]
I have trained the LPRNet on synthetic images for Indian number plates.
The CTC loss has come up to 3 and not decreasing beyond that. 

When I test it on synthetic images, accuracy is around 60 percent.
And when I test it on natural images, accuracy is just 2 percent

Any suggestions on how I can improvise the model?

This is one of the synthetic images that I have trained the model upon

![000003](https://user-images.githubusercontent.com/58128462/103064925-3d16f080-45db-11eb-96d7-7c6421eed6b3.png)

This is one of the natural images that I have tested upon

![KA01HH2975](https://user-images.githubusercontent.com/58128462/103065069-aac31c80-45db-11eb-92c7-193b9940b019.png)

**Comment by DKprojektai** [comment]
Please try [www.videoanalitika.lt/en](https://www.videoanalitika.lt/en)

**Comment by Pked01** [comment]
I am facing same issue, training LPR model on English alphabet give very low accuracy

**Comment by Ilya-Krylov** [comment]
We have an incoming [PR](https://github.com/openvinotoolkit/training_extensions/pull/464) with alphanumeric text recognition model training code on PyTorch. Tensorflow-based models are supported anymore.
